geth/node/manager.go

```diff
@@ -518,3 +519,59 @@ func (m *NodeManager) isNodeAvailable() error {
 
 	return nil
 }
+
+// Sync waits until blockchain synchronization and returns.
+func (m *NodeManager) Sync(timeout time.Duration) error {
+	// We need to have a larger timeout than ticker delta here
+	// unless we use zero which means infinite timeout.
+	if timeout < time.Second*2 && timeout != 0 {
+		return errors.New("Sync timeout can only be zero (infinite) or at least two seconds")
+	}
+	// Don't wait for any blockchain sync for the
+	// local private chain as blocks are never mined.
+	if m.config.NetworkID == params.StatusChainNetworkID {
+		return nil
+	}
+	if m.lesService == nil {
+		return errors.New("LightEthereumService is nil")
+	}
+	return m.sync(timeout)
+}
+
+func (m *NodeManager) sync(timeout time.Duration) error {
+	if timeout == 0 {
+		// Wait for a year (infinite).
+		timeout = time.Hour * 8765
+	}
+
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
Can you please extract `1 * Second` as a constant (`tickerResolution`) and re-use it in the timeout check in `Sync`.

Reply:
To be honest, I don't know why it's `1 * time.Second`. `time.Second` should already suit well. :smile:

And consider it done.